modules/services/avizo.nix

```diff
@@ -0,0 +1,74 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let cfg = config.services.avizo;
+in {
+
+  meta.maintainers = [ hm.maintainers.pltanton ];
+
+  options.services.avizo = {
+    enable = mkOption {
+      default = false;
+      description = ''
+        Whether to enable avizo.
+        Avizo is a simple notification daemon, mainly intended to be used for multimedia keys for example with Sway.
+      '';
+    };
+    config = mkOption {
+      type = types.attrs;
```

Comment:
Please make this more specific using [pkgs.formats.ini](https://nixos.org/manual/nixos/unstable/#sec-settings-nix-representable)

Reply:
Fixed